lib/phoenix_test/playwright/message.ex

```diff
@@ -0,0 +1,45 @@
+defmodule PhoenixTest.Playwright.Message do
+  @moduledoc """
+  Parse playwright messages.
+  A single `Port` message can contain multiple Playwright messages and/or a fraction of a message.
+  Such a message fraction is stored in `bufffer` and continued in the next `Port` message.
```

Comment:
Tiny typo. Thank for the comment! Helpful to know why we need this. 

```suggestion
  Such a message fraction is stored in `buffer` and continued in the next `Port` message.
```